sbin/start-worker.sh

```diff
@@ -47,7 +47,7 @@ if [[ $# -lt 1 ]] || [[ "$@" = *--help ]] || [[ "$@" = *-h ]]; then
   pattern+="\|Registered signal handler for"
 
   "${SPARK_HOME}"/bin/spark-class $CLASS --help 2>&1 | grep -v "$pattern" 1>&2
-  exit 1
+  [[ $# -lt 1 ]] && exit 1 || exit 0
```

Comment:
Note that,
- `sbin/start-worker.sh` with no argument exits with `1` in the same way.
- `sbin/start-worker.sh --help` exists with `0` after this PR.